### ISSUE TITLE
Vector3.set methods missing argument

### ISSUE THREAD

**Issue by Foorack** [zz-roba/UdonPieCompiler#5]
# Problem
Trying to call `.set_x`, `.set_y` o `.set_z` on a UnityEngineVector3
Example code:
```python
pos = UnityEngineVector3.get_zero()
pos.set_y(SystemSingle(10)) # invalid
```
Throws:
```bash
Not Found UnityEngineVector3.set_y('SystemSingle',) instance function.
```

This seems to be because the udon_funcs_data.py is missing data of arguments on these 3 functions?...

![bild](https://user-images.githubusercontent.com/5008081/73274532-fe6b9100-41e5-11ea-8f46-4c5ce2325609.png)

Strangely, calling `.set_x()` without any argument actually works in Unity, and does not throw an error, and seems to leave the value intact. I therefore suspect set_x is actually just another name for get_x?
For now it is possible to work around this by instead constructing a new Vector3 with `ctor`.

**Comment by Foorack** [comment]
This seems to have nothing to do with UdonPie. `Vector3 Set X/Y/Z` seems to be broken in normal Udon as well.

**Comment by zz-roba** [comment]
Thank you for checking.
UnityEngineColor has a similar phenomenon ...

```py
('InstanceFunc', 'UnityEngineColor', 'get_r', ()): ('SystemSingle', 'UnityEngineColor.__get_r__SystemSingle'),
('InstanceFunc', 'UnityEngineColor', 'set_r', ()): ('SystemSingle', 'UnityEngineColor.__set_r__SystemSingle'),
('InstanceFunc', 'UnityEngineColor', 'get_g', ()): ('SystemSingle', 'UnityEngineColor.__get_g__SystemSingle'),
('InstanceFunc', 'UnityEngineColor', 'set_g', ()): ('SystemSingle', 'UnityEngineColor.__set_g__SystemSingle'),
('InstanceFunc', 'UnityEngineColor', 'get_b', ()): ('SystemSingle', 'UnityEngineColor.__get_b__SystemSingle'),
('InstanceFunc', 'UnityEngineColor', 'set_b', ()): ('SystemSingle', 'UnityEngineColor.__set_b__SystemSingle'),
('InstanceFunc', 'UnityEngineColor', 'get_a', ()): ('SystemSingle', 'UnityEngineColor.__get_a__SystemSingle'),
('InstanceFunc', 'UnityEngineColor', 'set_a', ()): ('SystemSingle', 'UnityEngineColor.__set_a__SystemSingle'),
```

**Comment by Foorack** [comment]
@zz-roba Hm, yes. Although I don't think this is something UdonPie can fix. Just have to wait for VRChat to fix it in Udon... 🤔
